### PR TITLE
[FIX] Fix sdk export pipeline.json

### PR DESCRIPTION
**Pull request by irexyc**
## Motivation

Fix sdk export pipeline.json. 

The error is due to line 249

### CODE REVIEW

File: mmdeploy/backend/sdk/export_info.py [hunk 1]
```diff
@@ -245,8 +245,9 @@ def get_pipeline(deploy_cfg: mmengine.Config, model_cfg: mmengine.Config,
     task = get_task_type(deploy_cfg)
     input_names = preprocess['input']
     output_names = postprocess['output']
-    if task == Task.CLASSIFICATION or task == Task.SUPER_RESOLUTION \
-            or Task.VIDEO_RECOGNITION:
+    if task in [
```
Comment (RunningLeon) [inline mmdeploy/backend/sdk/export_info.py:248]: why need to change here? the logic seems unchange.

Comment (irexyc) [reply mmdeploy/backend/sdk/export_info.py:248]: It should be `task == Task.VIDEO_RECOGNITION` not just `Task.VIDEO_RECOGNITION`

Comment (RunningLeon) [reply mmdeploy/backend/sdk/export_info.py:248]: ooops

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**